packages/shared/examples/range-plot/range-plot.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang='ts'>
+
+  import { VisXYContainer, VisBulletLegend, VisTooltip, VisAxis, VisScatter, VisLine } from '@unovis/svelte'
+  import { Scale, Scatter } from '@unovis/ts'
+  import { data, DataRecord, processLineData } from './data'
+
+  const height = 1600
+  const yScale = Scale.scalePoint([0, 800]).domain(data.map(d => d.occupation))
+  const lineData = processLineData(data)
+  const legendItems = [{ name: 'Women', color: '#FF6B7E' }, { name: 'Men', color: '#4D8CFD' }]
+  const style: React.CSSProperties = { width: '100%', height: height }
```

Comment:
Same comment from the angular example above. I would remove in favor of template declaration